components/ArticleList.jsx

```diff
@@ -0,0 +1,10 @@
+import { ArticleCard } from "./ArticleCard";
+export const ArticleList = ({ articles }) => {
+  return (
+    <ul className="article-list">
+      {articles.map((article) => {
+        return <ArticleCard key={article.article_id} article={article} />;
+      })}
+    </ul>
+  );
+};
```

Comment:
It maybe worth looking to see if you need this file, or could you grab the article data in "Article" and pass them down to individual article cards? Essentially, you could cut out the middle Component and get rid of the need to pass the articles down on props.